### DISCUSSION TITLE
Wrong config option in default config for eslint?

### DISCUSSION THREAD

**Discussion by l3d00m** [LazyVim/LazyVim#4663] [General]
In the eslint plugin, the following config is specified: https://github.com/LazyVim/LazyVim/blob/cb40a09538dc0c417a7ffbbacdbdec90be4a792c/lua/lazyvim/plugins/extras/linting/eslint.lua#L11

According to https://github.com/neovim/nvim-lspconfig/blob/master/doc/configs.md#eslint it should be `workingDirectory` (singular) tho. Am I seeing this right, that this might be a bug?

**Comment by dpetka2001** [comment]
Create a PR to fix that if you'd like.

**Reply by l3d00m** [reply]
https://github.com/LazyVim/LazyVim/pull/4664
